asab/web/authz/service.py

```diff
@@ -13,8 +19,7 @@
 
 asab.Config.add_defaults({
 	"authz": {
-		"oauth2_url": "",
-		"userinfo_endpoint": "/userinfo",
+		"public_keys_url": "http://localhost:8081/.well_known/jwks.json",  # If set to "", ID tokens are not validated
```

Comment:
This is not "optimal" config default.

Reply:
i can't think of any reasonable, "least astonishing" default so i'm setting it to empty string, thus forcing the user to either provide the `public_keys_url` or set the dev option `_disable_token_verifiction` to True.